examples/python-fraud-examples/transactions-between-users.py

```diff
@@ -0,0 +1,447 @@
+        edges = g.E().project('id', 'source', 'target', 'label') \
+            .by(__.id_()) \
+            .by(__.out_v().id_()) \
+            .by(__.in_v().id_()) \
+            .by(__.label()) \
+            .dedup() \
+            .to_list()
+
+        for edge in edges:
+            edge_element = {
+                'data': {
+                    'id': str(edge['id']),
+                    'source': str(edge['source']),
+                    'target': str(edge['target']),
+                    'label': edge.get('label', '')
+                }
+            }
+            elements.append(edge_element)
+
+        #cluster.close()
+        return elements
+    except Exception as e:
+        print(f"Something went wrong {e}")
+        traceback.print_exc()
+
+
+def all_transactions_by_user(g, user_name):
+    #Find all transactions initiated by a specific user
+    print("\nQUERY 1: Transactions initiated by " + user_name + ":")
+    results =  g.V().has("User", "name", user_name) \
+        .out("owns") \
+        .outE("Transaction") \
+        .as_("transaction") \
+        .inV() \
+        .values("accountId") \
+        .as_("receiver") \
+        .select("transaction", "receiver") \
+        .by("amount") \
+        .by() \
+        .toList()
+    for result in results:
+        print(f"Transaction Amount: {result['transaction']}, Receiver Account ID: {result['receiver']}")
+
+
+def aggregate_transaction_amounts(g):
+    # Query Example 2: Aggregate total transaction amounts for each user
+    print("\nQUERY 2: Total transaction amounts initiated by users:")
+    results = g.V().hasLabel("Account") \
+        .group() \
+        .by("accountId") \
+        .by(__.out_e("Transaction").values("amount").sum_()) \
+        .toList()
+
+    for result in results:
+        print(result)
+
+
+def transfers_to_user(g, user_name):
+    print("\nQUERY 3: Users who transferred greater than 100 to " + user_name + ":")
+    results = g.V().has("User", "name", user_name) \
+        .out("owns") \
+        .inE("Transaction") \
+        .has("amount", P.gte(100)) \
+        .outV() \
+        .in_("owns") \
+        .valueMap("name") \
+        .toList()
+
+    for result in results:
+        print(f"User: {result}")
+
+
+def list_user_properties(g, user_name):
+    # Query Example 4: List all properties of a specific user
+    print("\nQUERY 4: Properties of " + user_name + ":")
+    user_properties = g.V().has("User", "name", user_name).valueMap().next()
+
+    # Iterate and print properties
+    for key, value in user_properties.items():
+        print(f"{key}: {value[0]}")
+
+
+def transactions_between_users(g, user1, user2):
+    person_1_id = g.V() \
+        .has('User','name',user1) \
+        .id_() \
+        .next()
+    person_2_id = g.V() \
+        .has('User','name', user2) \
+        .id_() \
+        .next()
+    results = g.V(person_1_id).outE().otherV().bothE().otherV().inE().otherV().hasId(person_2_id).path().by(T.id).toList()
```

Comment:
`outE().otherV()` is same as just `out()`

Reply:
I think we need the outE().otherV() b/c of path()